### PR TITLE
Updated docs to fix the npm install command to github from bitbucket

### PR DESCRIPTION
**Pull request by hpoom**
#### What are the links to relevant tickets?

No relevant tickets. Just saw this move from Bitbucket to Github today and wanted to fix the docs
#### What does this PR do?

Updates the docs to be more accurate
#### What unit tests does this PR have?

None, it is only a documentation change
#### How should a developer review this?

Check for spelling mistakes and try out the 'npm install' command to make sure what is documented works
#### Any background context you want to provide?

No
#### Screenshots (if appropriate)

No

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [ ] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [ ] I've tested this cross browser for any visual changes.
- [ ] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.
#### Reviewer 1 @kevinhodges
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running succesfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!
#### Reviewer 2
- [ ] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [ ] I’ve witnessed the tests running succesfully.
- [ ] I’ve checked for appropriate test coverage.
- [ ] I’ve checked for coding anti-patterns.
- [ ] I've checked this work against the requirements of the Jira.
- [ ] I don’t believe this work introduces unnecessary technical debt.
- [ ] +1 from me!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +24,15 @@ For non Sublime development a bunch of other IDE plugins are available [here](ht
...
-query    -> payload
-reply    -> payload
-request  -> payload
-response -> payload
-log      -> system
-responseTime -> system
-system   -> error
-thirdPartySystem -> error
+
+		query    -> payload
```
Comment (lukefrake) [inline README.md:28]: How come you've indented these?

Comment (hpoom) [inline README.md:28]: It looked like the original author of the docs was trying to keep that text formatted. Also it was hard to read, so by indenting it is now code formatted and easy to read. Happy to take it out, I did almost put it into a table ;)

Comment (lukefrake) [inline README.md:28]: Yeah I agree looks better formatted, might be better just using back tickets a la -

```
reply    -> payload
request  -> payload
response -> payload
log      -> system
responseTime -> system
system   -> error
thirdPartySystem -> error
```

Comment (lukefrake) [inline README.md:28]: p.s I'm being very picky ;)

Comment (hpoom) [inline README.md:28]: Look at the raw. https://raw.githubusercontent.com/holidayextras/loggly-interface/fix-docs/README.md
I think you would agree as unformatted raw markdown it is easier to read with indentation.

Comment (lukefrake) [inline README.md:28]: Yeah I think it comes down to more opinion, so I'll :+1:

**Comment by lukefrake** [comment]
I can't edit the original comment to tick boxes for some reason, so I'll just say I'll :+1: as reviewer 2

**Comment by hpoom** [comment]
Merged :)

**Pull request merged**